web/models/user.ex

```diff
@@ -19,7 +19,7 @@ defmodule Pairmotron.User do
   @minimum_password_length 8
 
   @required_params ~w(name email)
-  @optional_params ~w(password password_confirmation active role_id)
+  @optional_params ~w(password password_confirmation active is_admin)
```

Comment:
Totally correct as just discussed, but the :update action uses the regular changeset, in which you can change the role_id. So a regular user can essentially update themselves so that they are an admin.